MkDocs/apm_8.8/docs/matrix.md

```diff
@@ -0,0 +1,100 @@
+# Asset Matrix
+
+Maximo HP Utilities includes matrix view for determing the distribution of assets based on scores and then take action to avoid imminent failures.
+
+In this exercise the Reliability Engineer uses HP Utilities to:
+	
+1. [View Matrix for Criticality and End of life, Criticality and Risk, Criticality and Health](#view-matrix-for-criticality-and-end-of-life-criticality-and-risk-criticality-and-health)
+2. [View the drilled in assets list and take action](#view-the-drilled-in-assets-list-for-one-distribution)
+3. [Config Matrix version](#config-matrix-version)
+
+
+## Pre-requisites 
+
+This lab requires the following
+
+- A working Maximo Application Suite (MAS) environment with Maximo Manage, or another Enterprise Asset Management (EAM) system, Maximo Health, Maximo Health and Predict – Utilities (HPU), and Maximo Optimizer installed
+- Make sure the exercise [Understand Health and Predict - Utilities Models](hpu_models.md) is done, so that Sample ST(Substation Transformer) assets have the EOL/Risk/Health scores
```

Comment:
Can you please include a link to the doc that explains how how they add the EOL/Risk/Health scores?   Or point to existing lab exercises if the doc doesn't exist.

Reply:
the second one `Make sure the exercise [Understand Health and Predict - Utilities Models](hpu_models.md) ` is the exercise for how to add scores by notebook.